packages/std/src/imports.rs

```diff
@@ -178,6 +182,27 @@ impl Api for ExternalApi {
         Ok(address.into())
     }
 
+    fn secp256k1_verify(
+        &self,
+        message_hash: &[u8],
+        signature: &[u8],
+        public_key: &[u8],
+    ) -> StdResult<bool> {
```

Comment:
`StdResult` wrapper is useless now. Do we go ahead and remove it? Or we keep it, just for API consistency?

Reply:
Let's remove it. The fewer errors the contract developer needs to handle, the better. We have many API calls that can only:
- Succeed
- Fail in VM and never return to contract
- Panic in contract

All the Storage API works like this.

Reply:
Hmmm, it seems `StdResult` is a good idea here, for `Api` compatibility; i.e. that way the `MockApi` impl can return errors, in contract testing land.

Not sure if we want that, though. Though it will ease development / debugging, it points to functionality that will never trigger in the real environment.

What do you think?

Reply:
Good point. I tend to favour panics in the `MockApi`. Those can easily debug and we optimize for the production use case where there is nothing the contract developer should or can handle.

Reply:
OK, going for that then.

I'll publish these latest changes in a new PR (`cosmwasm-crypto` package) again `main`. Besides api / contract testing, and some extra docs / comments, I would say, this is done!™️.

Reply:
Continued in #779.